release-notes/10.0/preview/preview3/api-diff/Microsoft.NETCore.App/10.0-preview3_System.Security.Cryptography.md

```diff
@@ -0,0 +1,39 @@
+# System.Security.Cryptography
+
+```diff
+  namespace System.Security.Cryptography
+  {
+      public abstract class SymmetricAlgorithm : System.IDisposable
+      {
++         public void SetKey(System.ReadOnlySpan<byte> key);
++         protected virtual void SetKeyCore(System.ReadOnlySpan<byte> key);
+      }
+  }
+  namespace System.Security.Cryptography.X509Certificates
+  {
+      public sealed class PublicKey
+      {
+-         public PublicKey(System.Security.Cryptography.Oid oid, System.Security.Cryptography.AsnEncodedData parameters, System.Security.Cryptography.AsnEncodedData keyValue);
++         public PublicKey(System.Security.Cryptography.Oid oid, System.Security.Cryptography.AsnEncodedData? parameters, System.Security.Cryptography.AsnEncodedData keyValue);
+-         public System.Security.Cryptography.AsnEncodedData EncodedParameters { get; }
++         public System.Security.Cryptography.AsnEncodedData? EncodedParameters { get; }
+      }
+      public class X509Certificate : System.IDisposable, System.Runtime.Serialization.IDeserializationCallback, System.Runtime.Serialization.ISerializable
+      {
+-         public virtual byte[] GetKeyAlgorithmParameters();
++         public virtual byte[]? GetKeyAlgorithmParameters();
+-         public virtual string GetKeyAlgorithmParametersString();
++         public virtual string? GetKeyAlgorithmParametersString();
++         public byte[] ExportPkcs12(System.Security.Cryptography.PbeParameters exportParameters, string? password);
++         public byte[] ExportPkcs12(System.Security.Cryptography.X509Certificates.Pkcs12ExportPbeParameters exportParameters, string? password);
+      }
+      public class X509Certificate2Collection : System.Security.Cryptography.X509Certificates.X509CertificateCollection, System.Collections.Generic.IEnumerable<System.Security.Cryptography.X509Certificates.X509Certificate2>, System.Collections.IEnumerable
+      {
++         public byte[] ExportPkcs12(System.Security.Cryptography.PbeParameters exportParameters, string? password);
++         public byte[] ExportPkcs12(System.Security.Cryptography.X509Certificates.Pkcs12ExportPbeParameters exportParameters, string? password);
+      }
++     public enum Pkcs12ExportPbeParameters
++     {
++     }
```

Comment:
This seems to have missed the enum members?

https://github.com/dotnet/runtime/blob/release/10.0-preview3/src/libraries/System.Security.Cryptography/ref/System.Security.Cryptography.cs#L2912-L2917

```C#
    public enum Pkcs12ExportPbeParameters
    {
        Default = 0,
        Pkcs12TripleDesSha1 = 1,
        Pbes2Aes256Sha256 = 2,
    }
```

Reply:
Ding ding ding, new bug found!